frontend/rust-lib/flowy-document2/src/entities.rs

```diff
@@ -23,7 +23,7 @@ pub struct CloseDocumentPayloadPBV2 {
   // Support customize initial data
 }
 
-#[derive(Default, ProtoBuf)]
+#[derive(Default, ProtoBuf, Debug)]
 pub struct ApplyActionPayloadPBV2 {
```

Comment:
what's the meaning of V2?

Reply:
In order to distinguish it from the previous version used by Lucas in Flutter